tests/test_tasks.py

```diff
@@ -101,17 +111,20 @@ def setUp(self):
         self.to_delete = [self.p1, self.p2, self.n1_1, self.photo]
 
     def tearDown(self):
+        self.testbed.deactivate()
+
+        # TODO(yaboo@): delete below
```

Comment:
Since I use testbed, just testbed.deactivate() is ok to delete db entries. That's why I add 'TODO(yaboo@): delete below' here. But I think it is better to delete this line now to avoid confuse.